### PR TITLE
add stack

### PR DESCRIPTION
**Pull request by gustavotiecker**
This PR includes the complete implementation of the Stack data structure. At the end of the file I include some lines that can be used to test the data structure.

### CODE REVIEW

**Comment by gustavotiecker** [comment]
@vbrazo Please, would you review my PR?

File: data_structures/Stack/stack.swift [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+stack.push(3)
+stack.count
+stack.peek
+stack.isEmpty
+
+print(stack.pop())
+print(stack.pop())
+print(stack.pop())
+stack.isEmpty
+stack.count
```
Comment (vbrazo) [inline data_structures/Stack/stack.swift:42]: Could we please at least print the output here?

Comment (gustavotiecker) [reply data_structures/Stack/stack.swift]: Sure! Print statements have been added.

Comment (gustavotiecker) [reply data_structures/Stack/stack.swift]: @vbrazo

**Review by vbrazo — APPROVED** [review]

**Pull request merged**